### PR TITLE
feat: isolation package umd build support 

### PR DESCRIPTION
**Pull request by noname0310**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Provides umd build for isolation package.

### CODE REVIEW

**Comment by noname0310** [comment]
recovered

**Comment by ZLY201** [comment]
Maybe we should add copyright to `umd` bundle.

```javscript
/*!
 * js-sdsl v4.1.5
 * https://github.com/js-sdsl/js-sdsl
 * (c) 2020-present ZLY201 <zilongyao1366@gmail.com>
 * MIT license
 */
```

File: conf/umd-banner.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<package> v<version>
+https://github.com/js-sdsl/js-sdsl
+(c) 2020-present ZLY201 <zilongyao1366@gmail.com>
```
Comment (ZLY201) [inline conf/umd-banner.txt:3]: Sorry, I got it wrong.

`(c) 2021-present ZLY201 <zilongyao1366@gmail.com>`

File: conf/umd-banner.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<package> v<version>
```
Comment (ZLY201) [inline conf/umd-banner.txt:1]: package name should be @js-dsl/xxx I think.

**Review by ZLY201 — APPROVED** [review]

**Pull request merged**